core/src/main/java/org/mapfish/print/test/util/ImageSimilarity.java

```diff
@@ -215,7 +216,7 @@ private File getRelatedFile(final String name) {
    *
    * @return a number between 0 and 10000 or Double.MAX_VALUE on images format error.
    */
-  private double calcDistance(final BufferedImage actual) {
+  public double calcDistance(final BufferedImage actual) {
```

Comment:
Since it was not exposed in previous version. Would make sense to make it package visible.

Reply:
Then I didn't succeed to using it in the tests :-(